<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Regression Suite" duration-ms="6210" started-at="2020-06-05T19:30:29 PDT" finished-at="2020-06-05T19:30:35 PDT">
    <groups>
      <group name="regression">
        <method signature="LoginTest.emptyUsernameLogin()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="emptyUsernameLogin" class="com.hrms.testcases.LoginTest"/>
        <method signature="LoginTest.invalidPasswordLogin()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="invalidPasswordLogin" class="com.hrms.testcases.LoginTest"/>
      </group> <!-- regression -->
    </groups>
    <test name="Regression Test" duration-ms="6210" started-at="2020-06-05T19:30:29 PDT" finished-at="2020-06-05T19:30:35 PDT">
      <class name="com.hrms.testcases.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="setUp" is-config="true" duration-ms="2741" started-at="2020-06-05T19:30:29 PDT" finished-at="2020-06-05T19:30:32 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="emptyUsernameLogin()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="emptyUsernameLogin" duration-ms="383" started-at="2020-06-05T19:30:32 PDT" finished-at="2020-06-05T19:30:32 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emptyUsernameLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="tearDown" is-config="true" duration-ms="146" started-at="2020-06-05T19:30:32 PDT" finished-at="2020-06-05T19:30:32 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="setUp" is-config="true" duration-ms="1616" started-at="2020-06-05T19:30:32 PDT" finished-at="2020-06-05T19:30:34 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="invalidPasswordLogin()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="invalidPasswordLogin" duration-ms="1138" started-at="2020-06-05T19:30:34 PDT" finished-at="2020-06-05T19:30:35 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error message text is not matched expected [Invalid credential] but found [Invalid credentials]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error message text is not matched expected [Invalid credential] but found [Invalid credentials]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at com.hrms.testcases.LoginTest.invalidPasswordLogin(LoginTest.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidPasswordLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginTest@5faeada1]" name="tearDown" is-config="true" duration-ms="146" started-at="2020-06-05T19:30:35 PDT" finished-at="2020-06-05T19:30:35 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testcases.LoginTest -->
    </test> <!-- Regression Test -->
  </suite> <!-- Regression Suite -->
</testng-results>
